app/views/users/bio.html.erb

```diff
@@ -0,0 +1,59 @@
+<div class="grid-container margin-top-10">
+  <div class="grid-row grid-gap">
+    <div class="grid-col-3 margin-right-5">
+      <% if @user.avatar.present? %>
+        <%= image_tag @user.avatar_s3_presigned_url(:thumb), alt: "user avatar for #{@user.full_name}", class: 'avatar-profile-photo' %>
+      <% else %>
+        <div class="profile-avatar-container bg-base-lightest radius-md cropper-image-placeholder">
+          <i class="fas fa-user empty-user-avatar text-base-lighter"></i>
+        </div>
+      <% end %>
+    </div>
+
+    <div>
+      <!-- User Name, Job Title, Project -->
+      <h2 class="margin-top-0"><%= @user.bio_page_name %></h2>
+      <% job_title = @user.alt_job_title || @user.job_title %>
+      <% if job_title.present? %>
+        <p class="margin-top-2"><%= job_title %></p>
+      <% end %>
+      <% if @user.project.present? %>
+        <p class="margin-top-2"><%= @user.project %></p>
+      <% end %>
+    </div>
+  </div>
+
+  <div class="grid-row grid-gap">
+    <!-- Work -->
+    <% if @user.work.present? %>
+      <div class="grid-col-12 margin-top-5">
+        <h3 class="font-sans-md text-bold">Work</h3>
+        <ul class="usa-list usa-list--unstyled">
+          <% @user.work.split(",").each do |work_item| %>
+            <li><%= work_item %></li>
+          <% end %>
+        </ul>
+      </div>
+    <% end %>
+
+    <!-- About -->
+    <% if @user.bio.present? %>
+      <div class="grid-col-12 margin-top-4">
+        <h3 class="font-sans-md text-bold">About</h3>
+        <div class="line-height-sans-3 usa-prose">
```

Comment:
```suggestion
        <div class="line-height-sans-3">
```
Remove the `.usa-prose` here to get rid of the column clipping
![Screenshot 2024-11-13 at 4 12 49 PM](https://github.com/user-attachments/assets/8b2aed0c-79bc-4968-adb0-09e09c57cbe3)